src/test/java/duke/task/TodoTest.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+
+    @Test
+    public void testStringConversion() {
+        assertEquals("[T][✘] run", new Todo("run").toString());
+    }
+
+    @Test
+    public void markAsDone_regularTask_success() {
```

Comment:
clear naming of test function!! (sorry I just keep complimenting your code but your functions and variables are always really well-named and I am very impressed!!!